### ISSUE TITLE
ICO files failing with "Cant load  from ICO" error

### ISSUE THREAD

**Issue by kartikrao** [imgproxy/imgproxy#1266]
Hello @DarthSim, 
ICO files are failing with a cant load error - `imgproxy v3.19.0`

Stack trace:

```
{
    "client_ip": "124.168.178.123",
    "error": "Can't load  from ICO",
    "level": "error",
    "method": "GET",
    "msg": "Completed in 2.434938ms /unsafe/plain/https://d1hblyu8vvgvkg.cloudfront.net/drive-prod-media/drive/image/public/cms/theme/favicon.ico",
    "request_id": "rOcaIfwVRvnQB6GqD8jmY",
    "stack": "/app/processing_handler.go:219 main.sendErrAndPanic\n/app/processing_handler.go:226 main.checkErr\n/app/processing_handler.go:479 main.handleProcessing\n/app/server.go:138 main.buildRouter.withCORS.func4\n/app/server.go:114 main.buildRouter.withDevRequestLimit.func5\n/app/server.go:209 main.withPanicHandler.func1\n/app/server.go:127 main.buildRouter.withMetrics.func6\n/app/router/router.go:103 github.com/imgproxy/imgproxy/v3/router.(*Router).ServeHTTP\n/usr/local/go/src/net/http/server.go:2939 net/http.serverHandler.ServeHTTP\n/usr/local/go/src/net/http/server.go:2010 net/http.(*conn).serve",
    "status": 500,
    "time": "2024-04-04T23:07:42Z"
}
```

The following source URL's have valid ICO files that work in a browser but do not work through IMGPROXY:
- https://d1hblyu8vvgvkg.cloudfront.net/drive-prod-media/drive/image/public/cms/theme/favicon.ico
- https://d1hblyu8vvgvkg.cloudfront.net/drive-prod-media/drive/image/public/cms/theme/cci.ico

Here is another ico file that surprisingly works:
- https://d1hblyu8vvgvkg.cloudfront.net/drive-prod-media/drive/image/public/cms/theme/do.ico

**Comment by DarthSim** [comment]
Hey @kartikrao!

Thanks for the report! Fixed in the `latest` build.

**Comment by dbackeus** [comment]
Can confirm that using the latest build fixed this issue. Thanks 👍

**Comment by kartikrao** [comment]
@DarthSim - Thank You
When can we expect a release ?

**Comment by DarthSim** [comment]
@kartikrao I want to add some more changes before the release. I guess, we'll release a new version next week.

**Comment by kartikrao** [comment]
Hi @DarthSim - I tested these changes and the .ICO files work, however I found a regression on latest, SVG styles no longer work consistently.

**Comment by DarthSim** [comment]
@kartikrao Please create a new issue with samples

**Comment by DarthSim** [comment]
The fix is released in `v3.24.0`
